pkg/cmd/cli/cmd/rollout/rollout.go

```diff
@@ -134,3 +135,20 @@ func NewCmdRolloutUndo(fullName string, f *clientcmd.Factory, out io.Writer) *co
 	cmd.Example = fmt.Sprintf(rolloutUndoExample, fullName)
 	return cmd
 }
+
+const (
+	rolloutStatusLong = `
+Watch the status of the latest rollout, until it's done.`
+
+	rolloutStatusExample = `  # Watch the status of the latest rollout
```

Comment:
It watches until the end of the rollout. @fabianofranz asked for a --watch flag that we will default to true to make it clearer: https://github.com/kubernetes/kubernetes/issues/30627